index.html

```diff
@@ -8,73 +8,76 @@
       rel="stylesheet"
       href="https://unpkg.com/boxicons@latest/css/boxicons.min.css"
     />
+
     <link rel="stylesheet" href="style.css" />
-    <link rel="stylesheet" href="./css/DesktopVersion.css">
+    <link rel="stylesheet" href="./css/DesktopVersion.css" />
     <title>First setup</title>
   </head>
 
   <body>
     <header>
-      <nav>
-        <ul>
-          <li class="nameJacK">Jack</li>
-          <li class="Imgnoe"><a href="#">Portfolio</a></li>
-          <li class="Imgnoe"><a href="#">About</a></li>
-          <li class="Imgnoe"><a href="#">Contact</a></li>
-          <li><img src="img/menu_FILL0_wght400_GRAD0_opsz48.png" class="ImgCodeOn" alt=""></li>
-        </ul>
+      <nav class="navbar">
+        <a href="#" class="logo"><h2>Jack</h2></a>
+        <div class="nav-links">
+          <ul>
+            <img src="img/closeTag.svg" class="closeHumberger" alt="" />
```

Comment:
- [ ] Please make sure to match the design that your **hamburger** menu is **white** instead of **black***. :ok: 

### Your hamburger
![Screenshot from 2023-05-22 18-41-02](https://github.com/tchasinga/Setup-first-mobile/assets/89228743/511037bc-95ba-4c6c-9e1a-5b2af7a9036b)



### The hamburger on the template

![Screenshot from 2023-05-22 18-42-48](https://github.com/tchasinga/Setup-first-mobile/assets/89228743/4f8685ee-6bd1-469f-86c1-21c7428dfc59)

Reply:
Please ensure that, once the mobile menu is open, when we click on any link, the menu is closed and we are directed to the corresponding section on the page,
==============================================================================
I really fail to do that so that mean you'll not approved my project even if I submit